### ISSUE TITLE
[salt-cloud lxc] too verbose, shows host: True multiple times when starting

### ISSUE THREAD

**Issue by arthurlogilab** [saltstack/salt#23658]
```
# salt-cloud -m map_lxc_test 
[INFO    ] salt-cloud starting
ursa:
    True
[INFO    ] Runner completed: 20150513160351788451
[INFO    ] Applying map from 'map_lxc_test'.
ursa:
    True
ursa:
    True
ursa:
    True
ursa:
    True
ursa:
    True
ursa:
ursa:
    True
    True
[INFO    ] Runner completed: 20150513160353309611
[INFO    ] Runner completed: 20150513160353257734
[INFO    ] Runner completed: 20150513160353276393
[INFO    ] Runner completed: 20150513160353644330
[INFO    ] Runner completed: 20150513160353582599
[INFO    ] Runner completed: 20150513160353531490
[INFO    ] Runner completed: 20150513160353476720
[snip]
# salt-cloud --versions-report
                  Salt: 2015.5.0
                Python: 2.7.6 (default, Mar 22 2014, 22:59:56)
                Jinja2: 2.7.2
              M2Crypto: 0.21.1
        msgpack-python: 0.3.0
          msgpack-pure: Not Installed
              pycrypto: 2.6.1
               libnacl: Not Installed
                PyYAML: 3.10
                 ioflo: Not Installed
                 PyZMQ: 14.0.1
                  RAET: Not Installed
                   ZMQ: 4.0.4
                  Mako: 0.9.1
       Apache Libcloud: 0.15.1
 Debian source package: 2015.5.0+ds-1trusty1

```

**Comment by kiorky** [comment]
this is due to the refactor of the runners, from @cachedout and @jacksontj.

I tried to lower down at max the output and catch most of them, but i think we cant do much more now...

**Comment by kiorky** [comment]
PRs: #23898 #23897

**Comment by rallytime** [comment]
@arthurlogilab This appears to be fixed on 2015.8.0. When you get a chance to update, can you confirm that this is fixed?

**Comment by rallytime** [comment]
Since we didn't hear back, I am going to close this. If this remains an issue, leave a comment and we will happily open this issue again. Thanks!
